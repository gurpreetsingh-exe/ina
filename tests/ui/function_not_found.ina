// COMPILE_FAIL

fn main() {
    f();
}
