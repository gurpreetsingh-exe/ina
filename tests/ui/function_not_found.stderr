error: function `f` is not found in this scope
   --> function_not_found.ray:4:5
    |
 LL |     f();
    |     ^^^ not found in this scope

